packages/yoroi-extension/app/components/layout/TopBarLayout.js

```diff
@@ -59,7 +59,7 @@ function TopBarLayout({
             boxShadow: !isRevampLayout && showAsCard === true && '0 2px 12px 0 rgba(0, 0, 0, 0.06)',
             borderRadius: !isRevampLayout && showAsCard === true && '8px',
             ...(showInContainer === true && {
-              background: 'var(--yoroi-palette-common-white)',
```

Comment:
Totally agree! In that case though, since it is used on the non-revamp layout, I left it as it was. Will be removes soonish, once revamp is activated for al users.